File: distributivo-frontend/src/app/feature/schedule/models/saveEventDTO.ts
import { DayWeekSelect } from "./dayWeekSelect";
import { RangeOptEvent } from "./rangeOptEvent";

export interface SaveEventDTO {
    schoolPeriod : string,
    classroom : string,
    date : string,
    hour : string,
    occupiedBy : string,
    rangeOpt : RangeOptEvent | null
}

File: distributivo-backend/src/main/java/edu/yavirac/distributivobackend/feature/schedule/SaveEventDTO.java
package edu.yavirac.distributivobackend.feature.schedule;

import java.time.LocalDate;

import lombok.Data;

@Data
public class SaveEventDTO {
    private Long id;
    private Integer schoolPeriod;
    private Integer classroom;
    private LocalDate date;
    private String hour; 
    private Integer occupiedBy;       
}
